README.md

```diff
@@ -39,3 +39,29 @@ See the Main branch in action here as an [Azure Static Web App](https://ashy-bay
 ### Storybook
 
 Storybook is also being built and deployed automatically on the Main branch, including a preview URL on each pull request. See it in action on this [Azure Static Web App](https://ambitious-stone-0033b3603.1.azurestaticapps.net/).
+
+## Manifests
+
+The new WebComponent based backoffice is made up of components which are the same building blocks that is available to all developers to do the same customisations.
+
+Component | Description
+ --- | ---
+CollectionBulkAction | When a collection of items are selected, this is the bulk action that is available such as Copy, Delete, Move etc
+CollectionView | A custom view for display a collection of items such as content items in list view.
+Dashboard | A dashboard that is visible in a specific section of the backoffice.
+DashboardCollection | A dashboard to display a specific entity from a store as a collection of items that use the CollectionView's and CollectionBulkAction's
+ExternalLoginProvider | Adds a custom login provider such as Github, Google etc to the user dialog that slides out when the user avatar is clicked in the top right corner.
+HeaderApp | The user avatar and the search icon in the top right are both 'Header Apps' where more can be added.
+PropertyAction | On a specific property editor on the label three dots are visible to show further actions such as Copy and Clear etc.
+PropertyEditorModel | x
+PropertyEditorUI | x
+Section | Sections or apps inside the backoffice such as Content, Media, Settings, Packages etc
+SectionView | Section views was previously known as Content/Context apps. Examples are Created Packages, Content Info etc
```

Comment:
This did most likely not have a name before, but I would say these where Tabs on a Section before.
See the Workspace View, as they are the new ContentApps